README.md

```diff
@@ -45,8 +45,91 @@ Installation will vary depending on whether you use the `yarn` or `npm` packages
 The steps are simple:
 
 1. Clone this repository
-2. [Install `yarn` on your system using these instructions](https://classic.yarnpkg.com/en/docs/install)
-3. Follow these steps next
+2. Install Yarn on your system following these instructions
+
+### Install Yarn on Windows Using NPM
```

Comment:
Add a section or bullet point about the docusaurus prerequisites with a link to https://docusaurus.io/docs/installation where they are listed.

Reply:
I used a Note option and bullet point to indicate the docusaurus requirements respectively. I have also implemented the other changes. Please check and let me know of any improvement i need to make.